bitcoin/src/util/base58.rs

```diff
@@ -171,14 +172,19 @@ pub fn from_check(data: &str) -> Result<Vec<u8>, Error> {
     if ret.len() < 4 {
         return Err(Error::TooShort(ret.len()));
     }
-    let ck_start = ret.len() - 4;
-    let expected = endian::slice_to_u32_le(&sha256d::Hash::hash(&ret[..ck_start])[..4]);
-    let actual = endian::slice_to_u32_le(&ret[ck_start..(ck_start + 4)]);
+    let check_start = ret.len() - 4;
+
+    let hash_check = &sha256d::Hash::hash(&ret[..check_start])[..4];
```

Comment:
Nit: putting `try_into().expect("...")` would bring the `expect` closer to the reason why it's correct so it would be easier to see.

Reply:
Cool, used as suggested.